dev-python/virtualenv-clone/files/virtualenv-clone-0.5.7-backport-pr79.patch

```diff
@@ -0,0 +1,65 @@
+https://bugs.gentoo.org/918079
+https://github.com/edwardgeorge/virtualenv-clone/pull/79
+
+From 028494ee84cf4626aab291c621b2b9ec23f8688a Mon Sep 17 00:00:00 2001
+From: matoro <matoro@users.noreply.github.com>
+Date: Mon, 18 Dec 2023 18:18:17 -0500
+Subject: [PATCH] Support Python 3.11, 3.12
+
+---
+ .travis.yml       | 2 ++
+ setup.py          | 2 ++
+ tests/__init__.py | 2 +-
+ tox.ini           | 2 +-
+ 4 files changed, 6 insertions(+), 2 deletions(-)
+
+diff --git a/.travis.yml b/.travis.yml
+index 5f7409c..0a7000a 100644
+--- a/.travis.yml
++++ b/.travis.yml
+@@ -8,6 +8,8 @@ python:
+   -   "3.8"
+   -   "3.9"
+   -   "3.10"
++  -   "3.11"
++  -   "3.12"
+ dist: xenial
+ sudo: true
+ 
+diff --git a/setup.py b/setup.py
+index 8af2026..24a515e 100644
+--- a/setup.py
++++ b/setup.py
+@@ -58,6 +58,8 @@ def run_tests(self):
+         "Programming Language :: Python :: 3.8",
+         "Programming Language :: Python :: 3.9",
+         "Programming Language :: Python :: 3.10",
++        "Programming Language :: Python :: 3.11",
++        "Programming Language :: Python :: 3.12",
+     ],
+     tests_require=test_requirements,
+     cmdclass={'test': ToxTest}
+diff --git a/tests/__init__.py b/tests/__init__.py
+index c9a52ca..14648f4 100644
+--- a/tests/__init__.py
++++ b/tests/__init__.py
+@@ -8,7 +8,7 @@
+ tmplocation = tempfile.mkdtemp()
+ venv_path = os.path.realpath(os.path.join(tmplocation,'srs_venv'))
+ clone_path = os.path.realpath(os.path.join(tmplocation,'clone_venv'))
+-versions = ['2.7', '3.4', '3.5', '3.6', '3.7', '3.8', '3.9', '3.10']
++versions = ['2.7', '3.4', '3.5', '3.6', '3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+ 
+ def clean():
+     if os.path.exists(tmplocation): shutil.rmtree(tmplocation)
```

Comment:
Do you actually need any other part than this? We really want to keep patches small.

Reply:
No, I just leave patches alone unless some part of them fails to apply.  Can change that.